openslides/core/static/js/core/core.js

```diff
@@ -176,6 +182,32 @@ angular.module('OpenSlidesApp.core', [
     });
 }])
 
+/* Converts number of seconds into string "hh:mm:ss" or "mm:ss" */
+.filter('osSecondsToTime', function() {
```

Comment:
Proposal. Put the function inside a list, so it is easier to add dependencies in the future.